### PR TITLE
Add roslyn 15.8 -> CLI 2.1.4xx trigger

### PR DESCRIPTION
**Pull request by nguerrera**

### CODE REVIEW

**Comment by nguerrera** [comment]
@eerhardt @mmitche @dotnet/dotnet-cli Please review. I've never done this before so I just guessed based on the entries nearby. I don't know how to test it.

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -1185,6 +1185,22 @@
...
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/roslyn/dev15.8/Latest_Packages.txt"
+      ],
+      "action": "cli-dependencies",
+      "delay": "00:05:00",
+      "actionArguments": {
+        "vsoSourceBranch": "release/2.1.4xx",
+        "vsoBuildParameters": {
+          "GITHUB_PULL_REQUEST_NOTIFICATIONS": "dotnet/dotnet-cli",
+          "GITHUB_UPSTREAM_BRANCH": "release/2.1.4xx",
+          "ROSLYN_VERSION_FRAGMENT": "dotnet/roslyn/dev15.7"
```
Comment (dagood) [inline Maestro/subscriptions.json]: Should this be `dev15.8`?

Comment (nguerrera) [reply Maestro/subscriptions.json]: yes

Comment (nguerrera) [reply Maestro/subscriptions.json]: fixed. thanks.

**Comment by nguerrera** [comment]
@dagood @mmitche I didn't see a PR from this. Is there a way to check if there is an issue with it?

**Comment by dagood** [comment]
Trigger worked, but the build didn't: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=1868869.

**Comment by nguerrera** [comment]
I think https://github.com/dotnet/cli/pull/9684 will fix that.

**Pull request merged**